backend/src/main/java/develup/api/DiscussionApi.java

```diff
@@ -72,6 +73,17 @@ public ResponseEntity<ApiResponse<DiscussionResponse>> updateDiscussion(
         return ResponseEntity.ok(new ApiResponse<>(response));
     }
 
+    @DeleteMapping("/discussions/{solutionId}")
+    @Operation(summary = "디스커션 삭제 API", description = "디스커션을 삭제합니다.")
+    public ResponseEntity<ApiResponse<Void>> deleteSolution(
+            @Auth Accessor accessor,
+            @PathVariable Long solutionId
+    ) {
+        discussionWriteService.delete(accessor.id(), solutionId);
+
+        return ResponseEntity.noContent().build();
+    }
+
```

Comment:
[Request Changes]

solutionId -> discussionId

```suggestion
    @DeleteMapping("/discussions/{discussionId}")
    @Operation(summary = "디스커션 삭제 API", description = "디스커션을 삭제합니다.")
    public ResponseEntity<ApiResponse<Void>> deleteSolution(
            @Auth Accessor accessor,
            @PathVariable Long discussionId
    ) {
        discussionWriteService.delete(accessor.id(), discussionId);

        return ResponseEntity.noContent().build();
    }

```

Reply:
반영했습니다! https://github.com/woowacourse-teams/2024-devel-up/pull/615/commits/77ea15caa10fe6a07efbf05e6f9923c69b467a40